rules/azure-naming-resources/rule.md

```diff
@@ -22,6 +22,10 @@ created: 2020-06-25T22:36:42.000Z
 archivedreason: null
 guid: ff423950-2e2e-46b2-bfa5-ef9d69e83774
 ---
+<iframe width="560" height="315" src="https://www.youtube.com/embed/sFpcaDicRlA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Hey Jonty,

While the iframe would work, it is better to use the cleaner markdown syntax to embed a video
See https://www.ssw.com.au/rules/rule#videos

```suggestion
`youtube: https://www.youtube.com/embed/sFpcaDicRlA`
```